ecl/hql/hqlutil.cpp

```diff
@@ -5524,17 +5524,36 @@ static bool matchOption(unsigned & cur, unsigned max, const char * buffer, unsig
 IHqlExpression * extractCppBodyAttrs(unsigned lenBuffer, const char * buffer)
 {
     OwnedHqlExpr attrs;
+    bool isInComment = false;
     unsigned prev = '\n';
+
     for (unsigned i=0; i < lenBuffer; i++)
     {
         char next = buffer[i];
         switch (next)
         {
+        case '*':
+            if ( '/' == prev )
```

Comment:
Minor: we do not normally put spaces after the ( or before the ) in an if condition.